webapp/components/form/AutocompleteDialog/store/useLocalState.js

```diff
@@ -24,6 +24,14 @@ export const useAutocompleteDialog = ({
   )
   const Actions = useActions({ setState, onItemSelect, onClose })
 
+  useEffect(() => {
+    setState(State.assocItems(items))
+  }, [items])
+
+  useEffect(() => {
+    setState(State.assocInputFields(inputField))
```

Comment:
should this be State.assocInputField instead of State.assocInputFields ?

Reply:
wow yes!